perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -111,6 +111,18 @@
 	 * @throw ResourceNotExistsException When Resource with <code>id</code> doesn't exist.
 	 */
 	/*#
+	 * Returns specified  Group-Resource attributes for selected Group and Resource.
+	 * If <code>workWithGroupAttributes == true</code> then also all non-empty Group attributes are returned.
```

Comment:
This is not true, since this method takes list of ``attrNames`` => it returns also empty attributes for both ``group_resource`` and ``group`` namespace if ``workWithGroupAttributes == true``. If passed list of names is empty, it should return all non-empty attributes. Please also fix spaces in above line. Thank you.